src/components/ReportingPayments.js

```diff
@@ -1,11 +1,80 @@
 import { Component, PropTypes, DOM as dom, createFactory } from 'react';
+import _ from 'ramda';
+import $ from '../money';
+import moment from 'moment';
 
 class ReportingPayments extends Component {
+  constructor(props) {
+    super(props);
+    this.transactionsByDate = this.transactionsByDate.bind(this);
+    this.paymentsByDate = this.paymentsByDate.bind(this);
+    this.renderPaymentsByDate = this.renderPaymentsByDate.bind(this);
+    this.renderHeader = this.renderHeader.bind(this);
+    this.renderRow = this.renderRow.bind(this);
+  }
+
+  dateRange() {
```

Comment:
This is just defaulting to last week for illustrative purposes--will be able to be selected from page in future. Is there a better way to do ranges with moment?